src/lib/components/Creatibutors/CreatibutorsModal.js

```diff
@@ -210,28 +248,20 @@ export class CreatibutorsModal extends Component {
         }
       });
 
+      let idString =[];
+      creatibutor.identifiers.forEach((i, idx) => {
+        idString.push(this.makeIdEntry(i));
+      });
+
       return {
         text: creatibutor.name,
-        value: orcid.identifier,
+        value: creatibutor.id,
```

Comment:
One query - I think this is safe as the value shouldn't propgate outside of the form(?) and the contributor.id is unique.

Reply:
Yes, that is correct, and should have been like this from the beginning :)